### ISSUE TITLE
Wrong sbt resolution does not include minor and artifacts fail to download

### ISSUE THREAD

**Issue by fmv1992** [scalameta/scalameta#2234]
Just letting you know that I had to use the following:

```
libraryDependencies += "org.scalameta" % "semanticdb-scalac-core_2.13.4" % "4.4.6",
```

Instead of:

```
libraryDependencies += "org.scalameta" %% "semanticdb-scalac-core" % "4.4.6",
```

For the artifacts to be downloaded and my build proceed.

**Comment by olafurpg** [comment]
Thank you for reporting! The semanticdb-scalac-core artifact is cross-published against the compiler version instead of the library version. The standard way to depend on this artifact is to use this sbt syntax
```scala
libraryDependencies += "org.scalameta"  "semanticdb-scalac-core" % "4.4.6" cross CrossVersion.full
```

**Comment by fmv1992** [comment]
@olafurpg  , thanks for your reply and work!

It might be relevant to say that this is not available for `2.13.4`, which is the version supported by the latest [Scala Native](http://www.scala-native.org/en/latest/user/sbt.html#scala-versions).

I was able to compile this locally from the `2.13.3` with zero modifications and it worked out of the box for `2.13.4`. You might want to publish this...

Feel free to close this issue.

**Comment by tgodzik** [comment]
We do publish for `2.13.4`, but we don't yet publish for Scala Native, which might be the issue here.
